analyses/molecular-subtyping-HGG/05-HGG-molecular-subtyping-fusion.Rmd

```diff
@@ -112,19 +117,54 @@ ntrk_summary_df <- ntrk_summary_df %>%
   replace(is.na(.), "None")
 
 head(ntrk_summary_df, n = 10)
+```
+## New subtype of HGG
+
+There is a new 2021 entity within HGGs called "Infant-type hemispheric glioma" (IHG). This HGG is cerebral (hemispheric), arises in early childhood, and is characterized by RTK (receptor tyrosine kinase) alterations, typically fusions, in the NTRK family or in ROS1, ALK, or MET.
+
+The subtypes are:
+
+IHG, NTRK-altered
+IHG, ROS1-altered
+IHG, ALK-altered
+IHG, MET-altered
+
+The gene fusion with _ROS1_, _ALK_ and _MET_ are added into the output
```

Comment:
OK, I just went through the above and I think we can cut this code a bit - what we want to do is change line 78 from:
```
```{r}
fusion_df <- read_tsv(file.path(input_dir, "hgg_fusion.tsv"))
```
```
to pull in this file: https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/dev/analyses/fusion-summary/results/fusion_summary_lgg_hgg_foi.tsv

It will be pulled locally for now, but during v12 release, we will put it in the data folder and update this code. Sorry for the inconvenience, but this way, you can just select the columns of interest here instead of making all of these separate data frames. Can you update the code to use this file? Essentially, you can probably just use this file, select the HGG cohort, select all of the fusions of interest (feel free to redo the FGFR1/NTRK code) and get a similar DF as `summary_df`